flowmachine/tests/test_displacement.py

```diff
@@ -26,54 +26,75 @@ def test_returns_expected_values(stat, sub_a_expected, sub_b_expected, get_dataf
     Test that we get expected return values for the various statistics
     """
     sub_a_id, sub_b_id = "j6QYNbMJgAwlVORP", "NG1km5NzBg5JD8nj"
+    rl = ModalLocation(
```

Comment:
It's fine to use the original `ModalLocation` here. But now that this is lifted to the level of the tests (rather than implicitly created inside the `Displacement` class) it makes the tests _slightly_ more awkward to read than necessary.

Just for readability, let's switch this reference location to a plain `daily_location`, e.g.
```
rl = daily_location("2016-01-01")
```
This will require changing the expected values in the `pytest.mark.parametrize` statement above.

In order to determine the correct values, it may be helpful to run the test once from the command and grep for the various lines that report the failure and the mismatched values so that you can simply copy and paste them into the code. For example:
```
$ pipenv run pytest -sv tests/test_displacement.py -k test_returns_expected_values 2>&1 | egrep "(test_returns_expected_values\[.*\]|assert)"
__________ test_returns_expected_values[median-130.327306-151.547503] __________
>       assert df.loc[sub_a_id].value == pytest.approx(sub_a_expected)
E       assert 398.64699488375 == 130.327306 ± 1.3e-04
__________ test_returns_expected_values[stddev-132.73844-131.026341] ___________
>       assert df.loc[sub_a_id].value == pytest.approx(sub_a_expected)
E       assert 182.885304966881 == 132.73844 ± 1.3e-04
____________ test_returns_expected_values[avg-176.90362-178.910994] ____________
>       assert df.loc[sub_a_id].value == pytest.approx(sub_a_expected)
E       assert 370.301009034846 == 176.90362 ± 1.8e-04
___________ test_returns_expected_values[sum-5307.1086-5188.418836] ____________
>       assert df.loc[sub_a_id].value == pytest.approx(sub_a_expected)
E       assert 11109.0302710454 == 5307.1086 ± 5.3e-03
___ test_returns_expected_values[variance-17619494.3698313-17167902.1823453] ___
>       assert df.loc[sub_a_id].value == pytest.approx(sub_a_expected)
E       assert 33447034.7728291 == 17619494.3698313 ± 1.8e+01
```

I also suggest doing this for the other tests below that use the original `ModalLocation` definition.